docker-compose.yaml

```diff
@@ -1,19 +1,20 @@
-version: '3.9'
+version: "3.3"
```

Comment:
oh do we really need to downgrade the schema version ?

Reply:
Oh!
Now that's what by mistake, going to fix it.